content/en/user-manual/graphics/lighting/shadows.md

```diff
@@ -54,6 +54,12 @@ The shadow mapping technique used by PlayCanvas has only finite resolution. Ther
 
 The shadow distance is the distance from the viewpoint beyond which directional light shadows are no longer rendered. The smaller this value, the crisper your shadows will be. The problem is that the viewer will be able to see the shadows suddenly appear as the viewpoint moves around the scene. Therefore, you should balance this value based on how far the player can see into the distance and generally what looks good.
 
+### Shadow Intensity
+
+The intensity of the shadow darkening where 1 is the darkest it can be in the scene and 0 having no effect.
```

Comment:
Seems a bit vague. I mean, what determines what's the darkest it can be in the scene? Maybe @mvaligursky can help on language here.

Reply:
maybe:
The intensity of the shadow, where 1 represents full intensity shadow cast by this light, and 0 represents no shadow.

Reply:
Yeah, sounds good to me.

Reply:
```suggestion
The intensity of the shadow, where 1 represents full intensity shadow cast by this light, and 0 represents no shadow.
```